xml/geo_concept_i.xml

```diff
@@ -228,15 +228,19 @@
    <term>Ticket Failover</term>
    <listitem>
     <para>
-     If the ticket gets lost, which means other booth instances do not hear
-     from the ticket owner in a sufficiently long time, one of the remaining
-     sites will acquire the ticket. This is what is called ticket failover. If
-     the remaining members cannot form a majority, then the ticket cannot fail
-     over (if managed by booth).
+     After you have initially granted an automatic ticket to a site, booth
+     takes over and manages the ticket automatically.
+     If the site holding a ticket should be out of service, the ticket is
+     automatically revoked after the expiry time.
+     If the remaining sites have quorum, the ticket will be granted to another
+     site (fail over). The resources that depend on that ticket fail over to
+     the new site that holds the ticket. The <literal>loss-policy</literal>
+     set within the constraint specifies what happens to the nodes that have
+     run the resources before.
```

Comment:
"The loss-policy set within the constraint[...]"  ->  this is not completely clear for me.. Perhaps, we could clarify this a little, like: "The loss-policy, which was defined according to the .... , ..." ?